### ISSUE TITLE
Accessibility - Individual Goods Screen - TalkBack does Not Notify User of Content Change

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#45]
Why when updating ‘Exploitation Type’ list does application not notify the user of TalkBack that the screen has been updated with different goods/more or fewer goods on the list?

**Comment by aquilent-dgemoets** [comment]
We should be able to add a notification if such a notification is supported by the Android platform.

**Comment by jfc3-dol** [comment]
Guess you never got to updating this, since TalkBack doesn't notify me that the list changed when choosing a different type of labor.

**Comment by aquilent-dgemoets** [comment]
Correct.  It is number two on the priority list at the moment and is expected to be in the final release.

**Comment by nmangu** [comment]
Issue AS-IS.

**Comment by jfc3-dol** [comment]
TalkBack was able to notify the user that the list changed when they picked one type of labor.
